### ISSUE TITLE
dm2-xnat-export.py SQL error

### ISSUE THREAD

**Issue by josephdviviano** [TIGRLab/admin#922]
```
clevis@tigrsrv:/archive/code/config$ dm2-xnat-extract.py SPINS
...
Traceback (most recent call last):
  File "/archive/code/datman/bin/dm2-xnat-extract.py", line 752, in <module>
    main()
  File "/archive/code/datman/bin/dm2-xnat-extract.py", line 205, in main
    process_session(session)
  File "/archive/code/datman/bin/dm2-xnat-extract.py", line 279, in process_session
    process_scans(xnat_project, session_label, experiment_label, data)
  File "/archive/code/datman/bin/dm2-xnat-extract.py", line 531, in process_scans
    dashboard.delete_extra_scans(session_label, scans_added)
  File "/archive/code/datman/datman/dashboard.py", line 234, in delete_extra_scans
    db.session.delete(db_scan)
  File "/opt/quarantine/python-extras/2.7.10/build/sqlalchemy/orm/scoping.py", line 157, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/opt/quarantine/python-extras/2.7.10/build/sqlalchemy/orm/session.py", line 1710, in delete
    raise exc.UnmappedInstanceError(instance)
sqlalchemy.orm.exc.UnmappedInstanceError: Class 'flask_sqlalchemy.BaseQuery' is not mapped
```

**Comment by tomwright01** [comment]
Knew there would be something. Fixed in commit 345c81fcf4f1b3c116792e4fb701c9b9538fd28b

**Comment by josephdviviano** [comment]
Thanks!

Joseph's Phone
